.bumpversion.cfg

```diff
@@ -61,3 +61,11 @@ serialize =
 [bumpversion:file:octavia-cli/install.sh]
 
 [bumpversion:file:octavia-cli/setup.py]
+
+[bumpversion:file:connector-builder/Dockerfile]
```

Comment:
add the module to so its version gets bumped with the others

Reply:
does this mean that to get a connector builder update, I have to upgrade my whole airbyte instance? in that case we should probably recommend that people develop lowcode connectors in a separate instance from their prod one. Just a note for the docs in the future